Team12CMPE131/CMPE131Project
# Pull request template

Description: 

Add a pull request template so reviewers know exactly what changes are made.

How to test:

On GitHub.com
- Open a pull request to this branch
- Make sure you see "description", "issues linked", and "how to test"
